### ISSUE TITLE
Lack of information on return Error

### ISSUE THREAD

**Issue by DaniloCouto** [ionic-team/ionic-cli#2573]
**Description:**
Command `ionic cordova run android --prod --device` not returning a readable error.

**Steps to Reproduce:**
I've just tried to run `ionic cordova run android --prod --device`.
`ionic cordova run android --device` works normally.

**Output:**
```
[08:23:38]  build prod started ...
[08:23:38]  clean started ...
[08:23:38]  clean finished in 6 ms
[08:23:38]  copy started ...
[08:23:38]  ngc started ...
[08:24:02]  ngc finished in 24.55 s
[08:24:02]  preprocess started ...
[08:24:02]  deeplinks started ...
[08:24:03]  deeplinks finished in 1.01 s
[08:24:03]  optimization started ...
[08:24:04]  copy finished in 25.79 s
[08:24:14]  optimization finished in 10.70 s
[08:24:14]  preprocess finished in 11.71 s
[08:24:14]  webpack started ...
[08:24:20]  webpack finished in 6.19 s
[08:24:20]  sass started ...
[08:24:20]  uglifyjs started ...
[08:24:22]  sass finished in 1.49 s
[08:24:22]  cleancss started ...
[WARN] Error occurred during command execution from a CLI plugin (@ionic/cli-plugin-cordova).
Error: [object Object]
```

**My `ionic info`:**
```
cli packages:

    @ionic/cli-utils  : 1.6.0 (D:\Estudos\AjudanteOldDragon\node_modules\ionic\n
ode_modules\@ionic\cli-utils)
    ionic (Ionic CLI) : 3.6.0 (D:\Estudos\AjudanteOldDragon\node_modules\ionic)

global packages:

    Cordova CLI : 7.0.1
    Gulp CLI    : CLI version 1.2.2 Local version 3.9.1

local packages:

    @ionic/app-scripts              : 1.3.0
    @ionic/cli-plugin-cordova       : 1.4.1
    @ionic/cli-plugin-gulp          : 1.0.2
    @ionic/cli-plugin-ionic-angular : 1.3.2
    Cordova Platforms               : android 6.2.3
    Ionic Framework                 : ionic-angular 3.5.3

System:

    Android SDK Tools : 24.4.1
    Node              : v7.8.0
    OS                : Windows 7
    npm               : 4.1.1
```

**Comment by DaniloCouto** [comment]
Solved on app-scripts 1.3.1

**Comment by dwieeb** [comment]
@DaniloCouto There are updates in a few of those packages.
